src/utils.ts

```diff
@@ -129,16 +134,72 @@ export const assertEOSError = async (
 				error.json.error.name === eosErrorName,
 				`Expected ${eosErrorName}, got ${error.json.error.name} instead.`
 			);
-			return;
+			if (furtherErrorCheck) {
+				furtherErrorCheck(error)
+			};
 		} else {
 			// Fail if error not thrown by EOS
 			assert.fail(
 				`Expected EOS error ${eosErrorName}, but got ${JSON.stringify(error, null, 4)} instead.`
 			);
 		}
+		return true
+	}
+	return false;
+};
+
+/**
+ * Asserts EOS throws an error and validates the error output name matches the expected `eosErrorName`
+ * @author Kevin Brown <github.com/thekevinbrown>
+ * @param operation Operation promise
+ * @param eosErrorName Expected EOS error name
+ * @param description Output message description
+ */
+export const assertEOSError = async (
+	operation: Promise<any>,
+	eosErrorName: string,
+	description: string
+) => {
+	if (assertExpectedEOSError(operation, eosErrorName, (_) => {})) {
```

Comment:
We should be able to remove the `(_) => {}` here, yeah?

Reply:
👍 Thanks, good suggestion @thekevinbrown I'm still getting used to the syntax of TS/JS and what I can leave out.